packages/jobs/lib/routes/tasks/putTask.ts

```diff
@@ -85,15 +86,21 @@ const validate = validateRequest<PutTask>({
                         additional_properties: z.record(z.string(), z.unknown()).optional()
                     })
                     .optional(),
-                output: jsonSchema.default(null)
+                output: jsonSchema.default(null),
+                stats: z
+                    .object({
+                        proxy_egress_bytes: z.number().default(0),
+                        proxy_ingress_bytes: z.number().default(0)
+                    })
+                    .default({ proxy_egress_bytes: 0, proxy_ingress_bytes: 0 })
```

Comment:
should be optional during transition no?

Reply:
that's what default does by setting a value if not defined